app/models/serial_work.rb

```diff
@@ -21,4 +23,16 @@ def delete_empty_series
       self.series.destroy
     end
   end
+
+  # Ensure series bookmarks are reindexed when a new work is added to a series
+  def update_series_index
+    if $rollout.active?(:start_new_indexing)
+      series.enqueue_to_index
+      series.bookmarks.each(&:enqueue_to_index)
+    end
+
+    unless $rollout.active?(:stop_old_indexing)
+      IndexQueue.enqueue_ids(Bookmark, series.bookmarks.pluck(:id), :background)
```

Comment:
Old indexing code looks good to me. It's similar to:

https://github.com/otwcode/otwarchive/blob/f274643b0dcca0f4a21f00d23c0f59b2b8096ffd/app/models/redis_search_index_queue.rb#L42

which predates the upgrade.